packages/web/components/ad-banner/ad-banner-content.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import '@testing-library/jest-dom';
```

Comment:
maybe put this in a setup file and then add the setup file to the jest.config

`setupFilesAfterEnv: ['<rootDir>/setup-tests.js'],`